cmd/kubeadm/app/preflight/checks.go

```diff
@@ -125,7 +125,7 @@ func (sc ServiceCheck) Name() string {
 
 // Check validates if the service is enabled and active.
 func (sc ServiceCheck) Check() (warnings, errorList []error) {
-	klog.V(1).Infoln("validating if the service is enabled and active")
+	klog.V(1).Infof("validating if %s service is enabled and active", sc.Service)
```

Comment:
possibly use?
`klog.V(1).Infof("validating if the %q service is enabled and active", sc.Service)`

Reply:
_looking up the difference between `%s` and `%q` in Go_

Reply:
my proposal adds the missing "the" and %q would double-quote the string.

Reply:
Done. Thanks for the review.